### PR TITLE
Add Ludum Dare 32 and MiniLD 56

### PR DESCRIPTION
**Pull request by dragonfi**
Basically copied the description and meta-data from 2014. I hope it's good.

### CODE REVIEW

File: jams/2015.json [hunk 1]
```diff
@@ -48,6 +48,28 @@
...
+      "name": "Mini Ludum Dare #56",
+      "start_date": "2015-01-31",
+      "end_date": "2015-02-02",
+      "description": "Mini Ludum Dares are here to fill the boring months without ordinary Ludum Dares. The host changes every time and so do the rules. It all depends on the host but MiniLDs are often relaxed compared to LDs.",
+      "local": false,
+      "url": "http://ludumdare.com/compo/2015/01/07/minild-56-pick-n-mix-2/",
+      "tags": [
+        "minild",
+        "ludum-dare"
+      ],
```
Comment (0x0961h) [inline jams/2015.json]: Delete this comma and everything will be okay.

Comment (dragonfi) [inline jams/2015.json]: Thanks for the quick reply. I was indeed sloppy on this one. Fixed it in meantime, checking it with a formatter, so it should be okay now.

**Comment by 0x0961h** [comment]
Thanks!

**Comment by leafo** [comment]
Deployed

**Pull request merged**